roles/debian/hypervisor/tasks/main.yml

```diff
@@ -103,13 +103,37 @@
     - cpufreq.default_governor=performance
     - intel_pstate=disable
     - nosoftlockup
-    - "rcu_nocbs={{ isolcpus }}"
     - rcu_nocb_poll
     - rcutree.kthread_prio=10
-    - "isolcpus=nohz,domain,managed_irq,{{ isolcpus }}"
     - skew_tick=1
     - tsc=reliable
 
+- name: "grub conf hypervisor isolcpus"
```

Comment:
It works, but I think in the future we should create a file inside /etc/default/grub.d/ instead of modifying directly /etc/default/grub. It is easier to revert.